### PR TITLE
1082 rh UI implement pages and logic from old rh

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Now that we've got the base for the new rh-ui, we need to implement the rest of the logic and pages that RH-UI needs.
# What has changed

- Added all templates from old rh-ui
- Launch a survey
- Added welsh translations
- Translation logic simplified compared to old RH
- Error handling copied over from rh-ui 
# How to test?

- Bring up Docker-dev and stop the old rh-ui
- Run this branch against docker-dev
- Launch a survey
- Check all error routes (404, invalid uac length, invalid uac, no uac entered, Connection error to rh-service)
- Check all page links work as expected.
- Check all welsh translations work
- Run unit tests 
# Links

[Trello](https://trello.com/c/BkGDjNN4/)

### CODE REVIEW

**Review by hbrace — COMMENTED** [review]
Not really a comment on your work, but would it make sense for the `make install` command to include the pipenv install and load_templates?

File: rh_ui/templates/info_pages/privacy-and-data-protection.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+{% block main %}
+<h1 class="ons-u-fs-xl">{{ _('Privacy and data protection') }}</h1>
+
+<p>{{ _('We will keep your information secure and confidential. This means that we will:') }}</p>
+{%- autoescape false -%}
+{{ onsList({
+            "itemsList": [
+                {
+                    "text": _('recognise that you are sharing personal information with us and treat it as confidential, as directed
+                        by the<a href="https://code.statisticsauthority.gov.uk/"> Code of Practice for Statistics</a>')
```
Comment (gavinleeedwards) [inline rh_ui/templates/info_pages/privacy-and-data-protection.html:29]: The links on this page should all open in a new tab.

**Review by gavinleeedwards — CHANGES_REQUESTED** [review]
Functionally, it all works good.  Just a few minor comments.  

The formatting on all pages is slightly different to the current rh-ui (comparing it to what we have in preprod).  The text and elements now stretch across the length of the page.  It's a definite difference but does is matter from a UI perspective?

File: rh_ui/logger_config.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
 from structlog.threadlocal import wrap_dict
```
[the remark this answers is not in the document]
Comment (ryangrundy7) [reply rh_ui/logger_config.py:7]: Good spot. I'll take a look into it

**Review by gavinleeedwards — APPROVED** [review]

**Review by hbrace — APPROVED** [review]

**Pull request merged**